### PR TITLE
Setup feature branch

### PR DESCRIPTION
**Pull request by Mar12358**
### Added features:
- Initialize Bookstore project
  - Setup React App.
  - Linters (ESLint and Stylelint)
  - Jest
- Usage of:
  - React Router and routes.
  - Components.
- Documentation (README)

### CODE REVIEW

File: src/components/BookFormCreator.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const BookFormCreator = () => {
+  return (
+    <>
+      <div>Book Form Creator Here</div>
```
Comment (souad988) [inline src/components/BookFormCreator.jsx]: very well done by creating all required components , but please add a form with  inputs and button so it renders as a real form but you don't have to make it functional for now

File: src/components/BookController.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const BookController = () => {
+  return (
+    <>
+      <div>Book Controllers here</div>
+    </>
+  );
+};
+
+export default BookController;
```
Comment (souad988) [inline src/components/BookController.jsx]: good job by creating a separate component for the book 👍🏼  , but it needs to 
Take the book data as a prop and renders the title and the author, and havea button for removing books
the button wouldn't remove the books yet, this is just a placeholder for now

File: src/components/BooksList.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+const BooksList = () => {
+  return (
+    <>
+      <div>Book List Here</div>
+      <BookController />
+    </>
+  );
+};
+
```
Comment (souad988) [inline src/components/BooksList.jsx]: this component is meant to receive a list of books and render books in a loop and pass them as prop to the Book component

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Mar12358 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good use of github workflows
Use react dom
No linter errors
Good README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @Mar12358 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


200w

## Highlights
:heavy_check_mark: Added form with inputs
:heavy_check_mark: Passed data as props


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**